game.sy

```diff
@@ -238,6 +270,95 @@ update :: fn delta: float -> void {
         p.try_grab' puck
     }
 
+    iter := 1000
```

Comment:
If we hit this cap - we crash - that's deliberate. I don't think we can allocate much more than 1000 collision steps without dropping frames.